### DISCUSSION TITLE
Does it work?

### DISCUSSION THREAD

**Discussion by neoacevedo** [luyadev/luya#2191] [Questions & Problems] [answered]
I only try to override the admin layout for a bit leaving the other things as is, and according with the luya cms admin module the layout is a public property just like in Yii, so I set the layout as:

```php
    'cmsadmin' => [
            'class' => 'luya\cms\admin\Module',
            'layout' => '@app\views\layouts\admin.php'
    ],
```

Taking in mind the documentation at https://api.luya.io/yii-base-module#$layout-detail  and in https://luya.io/guide/cms/cmslayouts.html#provide-cms-layouts the admin layout should be rendered but actually the layout is never rendered, it still renders the luya-cms-admin layout file.

**Comment by nadar** [comment, marked as answer]
**No, change the admin layout won't work, because this is intended.** We provided all the code you need from the vendor, we can update, change, improve and fix issues, when you start override the layout with the next update probably nothing won't work anymore. LUYA is not a boilerplate you install and modify, its really a living eco system straight from your vendor folder which can improve "overnight" with just a composer update command.
